CallsheetLocalizations/pl.xcloc/Localized Contents/pl.xliff

```diff
@@ -1132,26 +1209,32 @@
       </trans-unit>
       <trans-unit id="Released" xml:space="preserve">
         <source>Released</source>
+        <target>Wydane</target>
         <note>When a TV episode or movie was released in the past</note>
       </trans-unit>
       <trans-unit id="Released " xml:space="preserve">
         <source>Released </source>
+        <target>Wydane </target>
         <note/>
       </trans-unit>
       <trans-unit id="Releases" xml:space="preserve">
         <source>Releases</source>
+        <target>Będzie wydane</target>
         <note>When a TV episode or movie will be released</note>
       </trans-unit>
       <trans-unit id="Remove Pin" xml:space="preserve">
         <source>Remove Pin</source>
+        <target>Usuń pinezkę</target>
         <note/>
       </trans-unit>
       <trans-unit id="Remove pin" xml:space="preserve">
         <source>Remove pin</source>
+        <target>Usuń pinezkę</target>
         <note/>
       </trans-unit>
       <trans-unit id="Rent" xml:space="preserve">
         <source>Rent</source>
+        <target>Wynajem</target>
```

Comment:
"Wypożycz". W sumie terminologia jak z wypożyczalni wideo, ale tak to funkcjonuje.